test_todoapp.py

```diff
@@ -52,12 +41,11 @@ def test_index_page(mock_get_requests, client):
     assert "123456" in response.data.decode() 
 
 def mock_get_cards(url, params):
-       TRELLO_BOARD_ID = os.environ.get("trello_boardid")
-      # API_KEY = os.environ.get("trello_key")
-      # TOKEN = os.environ.get("trello_token")
-       if url == f"https://api.trello.com/1/boards/{TRELLO_BOARD_ID}/cards":
+       #TRELLO_BOARD_ID = os.environ.get("trello_boardid")
+
+       #if url == f"https://api.trello.com/1/boards/{TRELLO_BOARD_ID}/cards":
               response = Mock()
               response.status_code = 200
               response.json.return_value = thingstodo 
               return response
```

Comment:
Your test needs to be updated. Your app no longer calls `requests.get`

Nor does it expect an object with a status_code and json() function.

We suggest installing the `mongomock` library for a mock DB instance.